next/app/components/layouts/Header/Header.module.css

```
@@ -0,0 +1,23 @@
+.header {
+  display: flex;
```

アイコンとテキストの高さを揃えるため、`align-items`を使用しましょう！
```suggestion
  display: flex;
  align-items: center;
```